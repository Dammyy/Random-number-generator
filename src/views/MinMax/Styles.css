.min-max-card {
  width: 400px;
  margin: 20px auto;
  height: 90px;
  background: #fff;
  border-bottom: 1px solid #ccc;
}

.min-max-card-title {
  height: 40px;
  line-height: 40px;
  background: #e8e8e8;
  text-align: center;
  color: #606066;
  font-weight: bold;
  font-size: 19px;
}

.min-max-value {
  height: 30px;
  padding: 10px 0px;
  line-height: 30px;
  text-align: center;
  color: #000;
  font-weight: bold;
}
