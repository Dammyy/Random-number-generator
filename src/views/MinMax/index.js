import React from "react";
import './Styles.css'

const MinMax = props => {
  return (
    <div className="min-max-card">
      <div className="title-text">{props.title}</div>
      <div className="min-max-value">{props.value}</div>
    </div>
  );
};

export default MinMax
