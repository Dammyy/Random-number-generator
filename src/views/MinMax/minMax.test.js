import React from 'react';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import MinMax from'./'

configure({adapter: new Adapter()});
test('Renders', () => {
const props = {
  title: 'minMax',
  value: '08038475647'
}
  const wrapper = shallow(<MinMax {...props} />);
  expect(wrapper).toMatchSnapshot();
});
