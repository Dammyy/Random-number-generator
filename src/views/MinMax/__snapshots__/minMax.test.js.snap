// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MinMax
    title="minMax"
    value="08038475647"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="min-max-card-title"
        >
          minMax
        </div>,
        <div
          className="min-max-value"
        >
          08038475647
        </div>,
      ],
      "className": "min-max-card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "minMax",
          "className": "min-max-card-title",
        },
        "ref": null,
        "rendered": "minMax",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "08038475647",
          "className": "min-max-value",
        },
        "ref": null,
        "rendered": "08038475647",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="min-max-card-title"
          >
            minMax
          </div>,
          <div
            className="min-max-value"
          >
            08038475647
          </div>,
        ],
        "className": "min-max-card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "minMax",
            "className": "min-max-card-title",
          },
          "ref": null,
          "rendered": "minMax",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "08038475647",
            "className": "min-max-value",
          },
          "ref": null,
          "rendered": "08038475647",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
